commitizen/commands/commit.py

```diff
@@ -91,7 +91,11 @@ def __call__(self):
         signoff: bool = self.arguments.get("signoff")
 
         if signoff:
+            out.warn("signoff mechanic is deprecated, please use `cz commit -- -s` instead.")
             c = git.commit(m, "-s")
+
+        if self.arguments.get("extra_cli_args"):
```

Comment:
Do we need this if-else block? Could we handle the case in `git.commit()` when `extra_cli_args` is not provided?